test/cli/whole-program_test.py

```diff
@@ -210,6 +212,61 @@ def test_suppress_inline_project_builddir_j(tmpdir):
     os.mkdir(build_dir)
     __test_suppress_inline_project(tmpdir, ['-j2', '--cppcheck-build-dir={}'.format(build_dir)])
 
+@pytest.mark.parametrize("builddir", (False,True))
+def test_addon_rerun(tmpdir, builddir):
```

Comment:
If we are adding new Python tests it would make sense to use the modern `tmp_path` instead.

The main difference is that you use '/' to concat paths and you might need to explicitly convert it to `str` in some cases. Otherwise not much is different. See 0a36cdd493e50c4d33182457f55e51f0f087e523 for some tests I added recently.

Reply:
done